search.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+require_once('root.inc.php');
+require_once($ROOT . 'includes/autoload.php');
+
+$vehicleData = null;
+if (isset($_POST['search'])) {
+  $vin = stripslashes($_POST['vin']);
```

Comment:
stripslashes may be outdated

echo htmlspecialchars($inputValue, ENT_QUOTES, 'UTF-8');